superbit_lensing/medsmaker/scripts/process_mocks.py

```diff
@@ -74,15 +79,19 @@ def main():
 
     # Make a mask.
     logprint('Making mask...')
-    bm.make_mask(overwrite=clobber, mask_name='forecast_mask.fits')
+    bm.make_mask(mask_name='forecast_mask.fits')
```

Comment:
This location in `medsmaker_mocks` is actually a good candidate for a refactoring. When Eric and I first developed this code, it was for the 2019 flight data which had not yet been reduced, so we had to make a mask and weight file as well. Since then, `medsmaker.py` has been de-scoped, and all `make_mask` really does is set attributes.